src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -16,6 +16,11 @@
 
 /** Offers useful methods to handle {@link Quantity}s */
 public class QuantityUtil {
+
+  private static final double DEFAULT_ABSOLUTE_TOLERANCE = 0d;
+  private static final double DEFAULT_RELATIVE_TOLERANCE = 0.00000001d;
+  private static final double DEFAULT_ANGLE_TOLERANCE = 0.001d;
```

Comment:
Why did you choose different tolerances here? I would have assumed, that all methods are default very strict (not accepting any tolerances).

Reply:
I didn't have any base except for the angle, where I used the one from the tests, so I used 0 for absolute tolerance as this would mimic complete equality the best. If all are set to 0, should we keep the three tolerances or combine them into one, in case that we want to change one but not the others in the future?

Reply:
I would join them, as this is what the "original" equal methods do.